File: ambeth/Ambeth.Merge/ambeth/merge/ICUDResultPreprocessor.cs

namespace De.Osthus.Ambeth.Merge
{
    public interface ICUDResultPreprocessor
    {
        ProceedWithMergeHook GetProceedWithMergeHook();

        void CleanUp(ProceedWithMergeHook proceedWithMergeHook);

        // Returns null, if preprocessing has not yet finished
        bool? GetPreprocessSuccess(ProceedWithMergeHook proceedWithMergeHook);
    }
}

File: jambeth/jAmbeth.Merge/src/main/java/de/osthus/ambeth/merge/ICUDResultPreprocessor.java
package de.osthus.ambeth.merge;

public interface ICUDResultPreprocessor
{
	ProceedWithMergeHook getProceedWithMergeHook();

	void cleanUp(ProceedWithMergeHook proceedWithMergeHook);

	// Returns null, if preprocessing has not yet finished
	Boolean getPreprocessSuccess(ProceedWithMergeHook proceedWithMergeHook);
}
